src/Motor.Extensions.Hosting.RabbitMQ/LogEvents.cs

```diff
@@ -4,7 +4,7 @@ namespace Motor.Extensions.Hosting.RabbitMQ
 {
     public static class LogEvents
     {
-        public static readonly EventId UnexpectedErrorOnConsume = new EventId(0, nameof(UnexpectedErrorOnConsume));
-        public static readonly EventId CriticalFailureOnConsume = new EventId(1, nameof(CriticalFailureOnConsume));
+        public static readonly EventId UnexpectedErrorOnConsume = new(0, nameof(UnexpectedErrorOnConsume));
```

Comment:
nit: as the event IDs are actually kinda meaningless, why not making this a bit more convenient:

```cs
private static readonly int _lastEventId = 0;

public static readonly EventId UnexpectedErrorOnConsume = new(_lastEventId++, nameof(UnexpectedErrorOnConsume));
```

Reply:
I will it remember.